sycl/doc/extensions/supported/sycl_ext_intel_esimd/examples/README.md

```diff
@@ -65,4 +66,50 @@ is to show the basic ESIMD APIs in well known examples.
           });
     });
     ```
-3) TODO: Add more examples here.
+3)  Calling ESIMD from SYCL using invoke_simd and Shared Local Memory (SLM) - ["invoke_simd_slm"](./invoke_simd_slm.md).
+  
+    Please see the full source code here: ["invoke_simd_slm"](./invoke_simd_slm.md)
+    ```c++
+     [[intel::device_indirectly_callable]] SYCL_EXTERNAL void __regcall invoke_slm_load_store(
+     local_accessor<int, 1> *local_acc, uint32_t slm_byte_offset, int *in, int *out,
+     simd<uint32_t, VL> global_byte_offsets) SYCL_ESIMD_FUNCTION {
+        esimd::simd<uint32_t, VL> esimd_global_byte_offsets = global_byte_offsets;
+        uint32_t local_acc_offset =
+        static_cast<uint32_t>(
+          reinterpret_cast<std::uintptr_t>(local_acc->get_pointer())) +
+      slm_byte_offset;
+      auto local1 = esimd::slm_block_load<int, VL>(local_acc_offset);
+      auto local2 = esimd::slm_block_load<int, VL>(local_acc_offset +
+                                               LOCAL_RANGE * sizeof(int));
```

Comment:
It may be better to use the new changes/APIs added by Gregory recently:
```suggestion
      auto local1 = esimd::block_load<int, VL>(*local_acc, slm_byte_offset);
      auto local2 = esimd::block_load<int, VL>(*local_acc, slm_byte_offset + LOCAL_RANGE * sizeof(int));
```

Reply:
oh yeah great point, thanks